Yatima/Graph/Graph.lean

```diff
@@ -261,7 +261,7 @@ partial def strongConnect (v : Vertex) : sccM (List $ List Vertex) := do
       pure $ scc::sccs
```

Comment:
Do we want to try using the new `DefinitionVal.all` field in Lean: https://github.com/leanprover/lean4/blob/09c4af26fce85559202a9c8a65837da88fefcd5e/src/Lean/Declaration.lean#L105 and comparing to see if our `scc` returns the same things?

Reply:
Testing the new API will require a chain of updates on other repos. Let's leave it for a future PR 👍🏼